src/label-helpers.ts

```diff
@@ -55,10 +60,10 @@ function isLabelable(element: Element) {
 }
 
 function getLabels(
-  container: Element,
+  container: Element | Document,
```

Comment:
Why would we pass a document here? That doesn't make much sense IMO.

Reply:
I have it now because in `getSuggestedQuery` function document is passed as container

Reply:
Ah, I didn't notice that is just the container not the target element :+1: